src/functions/functions.ts

```diff
@@ -63,9 +63,7 @@ function logMessage(message: string): string {
  * Defines the implementation of the custom functions
  * for the function id defined in the metadata file (functions.json).
  */
-if (typeof(CustomFunctionMappings) !== "undefined") {
-  CustomFunctionMappings.ADD = add;
-  CustomFunctionMappings.CLOCK = clock;
-  CustomFunctionMappings.INCREMENT = increment;
-  CustomFunctionMappings.LOG = logMessage;
-}
+CustomFunctions.associate("ADD", add);
```

Comment:
Is it possible add a JSDoc @link or @tutorial to this that points to the update in the docs for this sample?

Reply:
The changes are staged in this branch: https://github.com/OfficeDev/office-js-docs-pr/tree/jan-staging